bentoml/_internal/runner/container.py

```diff
@@ -32,59 +33,46 @@ def create_payload(
         data: bytes,
         meta: t.Optional[t.Dict[str, t.Union[bool, int, float, str]]] = None,
     ) -> Payload:
-        return Payload(data, dict(meta or dict(), container=cls.__name__))
+        return Payload(data, meta or {}, container=cls.__name__)
```

Comment:
we have something like this in the data container code, looks like this is already in `meta`?
```
ontainer_cls = DataContainerRegistry.find_by_name(
            str(payload_list[0].meta.get("container"))
        )
```